account_banking_payment_export/__openerp__.py

```diff
@@ -41,6 +41,7 @@
         'views/account_payment.xml',
         'views/payment_mode.xml',
         'views/payment_mode_type.xml',
+        'views/account_invoice.xml',
```

Comment:
Hm. That will probably break things if using l10n_be_invoice_bba which also does a similar thing.

https://github.com/odoo/odoo/blob/8.0/addons/l10n_be_invoice_bba/account_invoice_view.xml#L12

Not sure how to solve this...